net/ipcalc/Portfile

```diff
@@ -2,16 +2,16 @@
 
 PortSystem          1.0
 PortGroup           perl5 1.0
+PortGroup           github 1.0
 
-name                ipcalc
-version             0.41
-revision            6
+github.setup        kjokjo ipcalc 0.51
```

Comment:
can you add "github.tarball_from" with the appropriate setting (i.e., "releases" or "archive")?